sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample05_IdentityAndSharedAccessCredentials.md

```diff
@@ -0,0 +1,201 @@
+# Identity and Shared Access Credentials
+
+This sample demonstrates using credentials to authorize clients with the Event Hubs service.  In most scenarios, credentials provide a better approach to security than connection strings for production applications.  One of the key benefits is support for in-place rotation to update access policies without the need to stop applications.  Most credential sources also allow for more fine-grained access control, when compared with the shared key approach favored by connection strings.
```

Comment:
>  Most credential sources also allow for more fine-grained access control, when compared with the shared key approach favored by connection strings.

It's unclear if you are saying shared key is better or worse than connection strings here.

```suggestion
This sample demonstrates using credentials to authorize clients with the Event Hubs service.  In most scenarios, shared access credentials provide a better approach to security than connection strings for production applications.  One of the key benefits is support for in-place rotation to update access policies without the need to stop applications.  Most credential sources also allow for more fine-grained access control, when compared with the shared key approach favored by connection strings.
```